src/components/Layout/TopNav.tsx

```diff
@@ -166,29 +172,29 @@ function TopNav() {
                   className={checkWebuiUpdate.isFetching || webuiUpdateResult.isLoading ? 'text-highlight-1' : 'text-highlight-2'}
                   spin={checkWebuiUpdate.isFetching || webuiUpdateResult.isLoading}
                 />
-                <div className="flex ml-3">
+                <div className="flex ml-2.5">
```

Comment:
figma shows gap of 8px so it is ml-2

Reply:
Same comment as above, gap wasn't enough